### ISSUE TITLE
Päivämäärällä suodattaminen

### ISSUE THREAD

**Issue by ajrajala** [KohaSuomi/Koha#576]
### Mikä vikana?

Huomasin, että päivämäärällä suodattamisessa on joillakin Kohan sivuilla samankaltainen ongelma kuin [tässä tiketissä](https://github.com/KohaSuomi/Koha-22x/issues/186). Ainakin nidehaun tuloksissa, asiakashaun tuloksissa ja taustatöissä päivämäärä-sarakkeissa suodattaminen edellyttää kirjoittamaan päivämäärän muodossa vvvv-kk-pp vaikka päivämäärät on käyttöliittymässä esitetty muodossa pp.kk.vvvv.

Joillakin sivuilla suodatus kelpuuttaa päivämäärän muodossa pp.kk.vvvv (esim. tietueen Perustiedot-näytöllä ja Kausijulkaisutilausten haun tuloksissa).

**Comment by AnneliO** [comment]
Liittyisköhän tähän bugiin: https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=34226

**Comment by ajrajala** [comment]
Nopeasti vilkaisin, että bugitiketti oli pushattu 23.11 masteriin, mutta se ei tainnutkaan vielä suoraan tarkoittaa raportoitujen ongelmien korjaantumista 23.11 versiossa vaan tuohon liittyen on edelleen kuitenkin muita tikettejä kesken..?

**Comment by ajrajala** [comment]
Joo, testasin nextillä nide- ja asiakashaun tulosnäkymiä ja niissä näytti ainakin vaativan edelleen että päivämäärän syöttää muodossa vvvv-kk-pp. 🤔

**Comment by ajrajala** [comment]
Kirjaan ylös, että versiossa 24.05 edelleen sama tilanne ja koska tietueen Perustiedot-sivulla on muutettu nidetietojen hakemista, ei sielläkään enää pysty suodattamaan päivämäärällä muussa muodossa kuin vvvv-kk-pp.

**Comment by AnneliO** [comment]
Koha-Suomen palaveri 2.9.2024: Teetkö Anni tästä yhteisötiketin?

**Comment by ajrajala** [comment]
Nyt on yhteisössä [tiketti 37875](https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=37875).

**Comment by ajrajala** [comment]
Tekemäni tiketin jälkeen yhteisöön tehty toinen tiketti, joka jätetty voimassa olevaksi: https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=38110
